src/server/server.js

```diff
@@ -45,6 +45,7 @@ const startServer = (port, ssoEndpoint, apigeeEndpoint, basePath) => {
                     data => { res.end('You are now authenticated'); },
                     e => {
                         console.log('Error during authentication: ' + JSON.stringify(e));
+                        res.statusCode = 401;
```

Comment:
This is not necessarily a real 401 error, right? it may be also a server error. Do we want to make that distinction or just consider all errors as 401?